constants/accountStatuses.js

```diff
@@ -0,0 +1,7 @@
+export const accountStatuses = Object.freeze({
```

Comment:
I think better to name accountStatus. Because it is not array.

Reply:
if it's used as enum naming is okay